samples/drivers/CAN/mcp2515-dts.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Karsten Koenig
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&spi1 {
+	status = "ok";
+	cs-gpios = <&gpioa 4 GPIO_DIR_OUT>;
+	mcp2515@0 {
+		compatible = "microchip,mcp2515";
+		spi-port-name = "SPI_1";
+		spi-max-frequency = <1000000>;
+		int-gpios = <&gpioa 0 GPIO_INT_ACTIVE_LOW>;
+		status = "ok";
+		label = "CAN_1";
+		reg = <0>;
+	};
+};
+
+&can1 {
```

Comment:
can1 seems to only exists on boards, that already have other can hardware. For me it compiles on
 - stm32f072b_disco

but not on 
 - nucleo_f091rc
 - stm32_min_dev

Also the generated fixup `DT_CAN_1_BUS_SPEED` on stm32f072b_disco board points at the stm32 CAN driver.